tests/test_storage_validation.py

```diff
@@ -69,7 +69,6 @@ def test_extrakeys_not_allowed(storage_type):
         "redis",
         "opensearch",
         "rds-postgres",
-        "aurora-postgres",
```

Comment:
@lgarvey Covering `aurora-postgres` here which now requires `version` where the others don't seems unnecessary to me. What do you think?